src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -70,21 +72,32 @@ only  ‘per-user’ namespace strategy is supported;
 a user can simultaneously  run only 1 workspace with asynchronous storage
 
 To configure asynchronous storage for your workspace add the following to workspace configuration:
++
 [source,yaml]
 ----
 attributes:
  asyncPersist: 'true'
  persistVolumes: 'false'
 ----
 
-Aslo two new properties added to `che.properties` that can be used to configure behavior of a client like Che UserDashboard:
+Also, the two new properties were added to `che.properties` that can be used to configure behavior of a client like Che UserDashboard:
```

Comment:
Useless sentence.
```suggestion
```

Reply:
To add, I'd avoid using "the new X" in the docs, since eventually it'll no longer be new.

Reply:
@MichalMaler, please, remove the sentence.

+1 to Angel's remark -- that holds true everywhere.